wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,331 @@
+
+Over the years the osu! global rankings has used three different systems for ranking players:
+
+- [Ranked Score](/wiki/score#ranked-score) (July 2007 – 24 July 2012)
+- [ppv1](/wiki/Performance_points/ppv1) (24 July 2012 – 27 January 2014)
+- [ppv2](/wiki/Performance_points) (27 January 2014 – Present)
+
+Because of this, this list will only include players who reached Rank #1 on the main leaderboard that was used at the time of reaching said rank. *Italicized* dates and numbers indicate that the corresponding player are continuing their reign as osu!'s Rank #1 player either from the previous year or onto the following year.
+
+## 2007
+
+*Note: During the period where Ranked Score was used, Rank #1 would change frequently, thus the earlier listings may use months where exact dates aren't available.*
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-10-11 | 2007-10-15 | [\[1\]](https://web.archive.org/web/20071011215722/http://osu.ppy.sh/index.php?p=player) |
+| ![][flag_AU] [eyup](https://osu.ppy.sh/users/88) | 2007-10-15 | 2007-10-24 | [\[1\]](https://osu.ppy.sh/community/forums/topics/342) |
+| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-10-24 | 2007-11-17 | [\[1\]](https://web.archive.org/web/20071024063540/http://osu.ppy.sh/index.php?p=player) |
+| ![][flag_NZ] [Echo](https://osu.ppy.sh/users/431) | 2007-11-17 | 2007-11-23 | [\[1\]](https://osu.ppy.sh/community/forums/topics/963) |
+| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-11-23 | 2007-12-24 | [\[1\]](https://web.archive.org/web/20071024063540/http://osu.ppy.sh/index.php?p=player) |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 2007-12-24 | *2008-09-07* | [\[1\]](https://web.archive.org/web/20071224062629/http://osu.ppy.sh/index.php?p=player) |
+
+## 2008
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | *2007-12-24* | 2008-09-07 | [\[1\]](https://web.archive.org/web/20071224062629/http://osu.ppy.sh/index.php?p=player) |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | 2008-09-07 | 2008-09-07 | [\[1\]](https://osu.ppy.sh/community/forums/topics/5999) |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 2008-09-07 | 2008-09-26 | [\[1\]](https://osu.ppy.sh/community/forums/topics/5999) [\[2\]](https://web.archive.org/web/20080913072815/http://osu.ppy.sh/index.php?p=player) |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2008-09-26 | 2008-11-23 | [\[1\]](https://osu.ppy.sh/community/forums/topics/6853?start=54620)  |
+| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | 2008-11-23 | 2008-12-20 | [\[1\]](https://osu.ppy.sh/community/forums/topics/7797) |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2008-12-20 | *2009-03-05* | [\[1\]](https://osu.ppy.sh/community/forums/topics/8415)  |
```

Comment:
All foreign links should use title text to indicate the name of the website; i added most of them in for you
```suggestion
| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | *2007-12-24* | 2008-09-07 | [\[1\]](https://web.archive.org/web/20071224062629/http://osu.ppy.sh/index.php?p=player "Internet Archive — osu!") |
| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | 2008-09-07 | 2008-09-07 | [\[1\]](https://osu.ppy.sh/community/forums/topics/5999) |
| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 2008-09-07 | 2008-09-26 | [\[1\]](https://osu.ppy.sh/community/forums/topics/5999) [\[2\]](https://web.archive.org/web/20080913072815/http://osu.ppy.sh/index.php?p=player "Internet Archive — osu!") |
| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2008-09-26 | 2008-11-23 | [\[1\]](https://osu.ppy.sh/community/forums/topics/6853?start=54620)  |
| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | 2008-11-23 | 2008-12-20 | [\[1\]](https://osu.ppy.sh/community/forums/topics/7797) |
| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2008-12-20 | *2009-03-05* | [\[1\]](https://osu.ppy.sh/community/forums/topics/8415)  |
```

Reply:
If anything the text should be indicating the name of the post/article/video, rather than the website it's on. Doing it this way makes very little sense to me.

Reply:
[its an ASC thing](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#external-links)™

Reply:
I still find it wrong to prioritise the website name rather than the source name, especially when it comes to citations rather than just some external link. Since it's on the ASC though, I suppose the best compromise would be doing something like "Site - Source title". Otherwise, it's very difficult to understand where the link actually takes you to. I'll get to work on that later, but I do want to hear what you think too.

Reply:
Well, the alt text for links isn't really meant to provide any "description" of what the link actually is, as that should be the job of title text.

There was a [discord convo](https://discord.com/channels/188630481301012481/218677502141399041/280008835207397387) abt this subject a while back (mega had mixed up "alt text" and "title text" here btw)